aggregate/housing_production/change_in_units.py

```diff
@@ -149,6 +150,11 @@ def clean_jobs(df):
 
     return df
 
+def rename_col(cols) -> List:
```

Comment:
This seems like a fine solution for this indicator. I changed the strings in the rename call in `pivot_and_flatten_index` as I think it's nice to have the entire indicator label hard-coded somewhere but this is fine as well

Reply:
yeah the issue here is the denom then get left out without the token. I don't see a clever way to add that for the moment